src/content/docs/en/pages/main-menu/reference/marketplace/marketplace-seller-guide.mdx

```diff
@@ -1,141 +1,213 @@
+<Button href="https://www.azion.com/en/pricing/" text="go to Pricing page" variant="secondary"> 
 
-Available for customers with current licenses purchased via other channels.
+</Button>
 
-BYOL (Bring Your Own License) solutions let you run software on Azion Edge Platform while using licenses purchased directly from the seller. Azion only charges customers for the Azion services costs, giving them the flexibility to purchase and manage their own licenses.
+### Pricing
 
-#### C. Software Subscription
+When you submit a product to Azion Marketplace, you define the price of the product and the terms and conditions of use. When a buyer subscribes to a product offering, they agree to the pricing and terms and conditions set for it.
 
-With the Software Subscription model you will use Azion Marketplace to charge for the usage of your product. You can choose between two different options, Pay As You Go (PAYG) or Saving Plans.
+You can create products with:
 
-**Pay As You Go**
+- A standard price list with a general end-user license agreement (EULA).
+- Private offers for individual customers with special pricing and EULAs are usually needed by large enterprises or requests for proposals (RFPs).
 
-In a Pay As You Go model Azion Marketplace bills your customers based on the metering records of usage of your software and the price for that usage, established by you. It is a model with no commitment and the usage is calculated and billed monthly.
+You can also choose between two different options, [Pay As You Go](#pay-as-you-go) or [Saving Plans](#saving-plans).
 
-For ACP-based products, Azion Marketplace will bill your customer based on the metering records available in the Azion Edge Platform. The available metering records are:
+:::note
+All pricing is based on US dollars (USD).
+:::
 
-* Number of deployed instances
-* Number of registered users
-* Requests
-* Compute Time
-* Data Transfer
+#### Pay As You Go
 
-For SaaS-based products, Azion Marketplace bills your customers based on the metering reported by your SaaS applying the price list established by you.
+By using the **Pay As You Go (PAYG)** model, Azion Marketplace bills your buyers based on the metering records of usage of your software and the price for that usage, established by you. It's a model with no commitment and the usage is calculated and billed monthly.
 
-The Azion Marketplace Metering Service requires that your software report usage every hour, recording the customer usage for the hour. If there is a failure in the transmission or receipt of metering service records, Azion will be unable to bill for such usage. You are responsible for ensuring the successful receipt of metering records.
+The available metering records in Azion Marketplace are:
 
-You can use any metering records that suit your SaaS business model, regarding that you have established the price list for it and your SaaS reports the usage correctly.
+- Number of deployed instances
```

Comment:
não é a partir de 3 que vai ponto final?

Reply:
No Style Guide aparece que "The most common rule we use is to end items in a list with more than three words with a full stop", mas daí por ser uma lista de ítens que a maioria é menos que 3 palavras, fiquei na dúvida...

Na MS, consegui essa recomendação: "Don’t use a period at the end of list items unless they’re complete sentences, even if the complete sentence is very short.", talvez cabe nessa... https://learn.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation

Reply:
esse caso parece caber na 2a, mesmo!